src/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -645,6 +645,18 @@ ${ flowData.code }
 
 	}
 
+	getSubgroupSize() {
+
+		return this.getBuiltin( 'subgroup_size', 'subgroupSize', 'u32', 'attribute' );
```

Comment:
I think we could call `enableSubGroups()` here and `getSubgroupIndex()`? to add directive automatically..

I think `enableDirective()` should use `Set` classe instead of `Array`.

Reply:
That sounds good by me, could probably also do the same for enableExtension if we want default behavior to be the same across extensions ( i.e enable or require )